pkg/cluster/generateconfig.go

```diff
@@ -177,7 +177,6 @@ func (m *manager) generateInstallConfig(ctx context.Context) (*installconfig.Ins
 					CloudName:                azuretypes.CloudEnvironment(m.env.Environment().Name),
 					OutboundType:             azuretypes.LoadbalancerOutboundType,
 					ResourceGroupName:        resourceGroup,
-					ARO:                      true,
```

Comment:
sorry for my ignorance, why is this going away? Is this related to the 'tags aro' coming from the installer (I think?)

Reply:
Yes, thanks to @m1kola who got the [aro build tag upstreamed](https://github.com/openshift/installer/pull/4843/files#diff-31eccc942d872dd4ac7f4471915aa73ed12857a44e0cfa40e19b8dbc32cf557b).